### ISSUE TITLE
Sheet pops in without animation when using css driver

### ISSUE THREAD

**Issue by hackily** [tamagui/tamagui#1974]
### Current Behavior

Sheet pops in without using animation when using css driver. The animation appears to work fine when the sheet is closed.

https://github.com/tamagui/tamagui/assets/8202335/728c6ef6-1be4-4c7e-8c24-7991dcdef0be

### Expected Behavior

Sheet should both pop in, and pop out with an animation when using css driver

### Tamagui Version

```markdown
1.79.12
```

### Platform (Web, iOS, Android)

```markdown
Web
```

### Reproduction

```markdown
https://tamagui.dev/docs/components/sheet/1.59.0
```

**Comment by natew** [comment]
@alitnk another good one to check, likely not too hard, it was working a few weeks ago so a test would be nice.

**Comment by alitnk** [comment]
Should be fixed by #2012
